src/components/MeasurementOption/MeasurementOption.js

```diff
@@ -114,6 +114,51 @@ class MeasurementOption extends React.Component {
     this.setState(state => ({ isEditing: !state.isEditing }));
   };
 
+  renderScaleInput = (type, val) => {
+    /**
+     * There is a bug with Firefox 69 where after onFocus, it calls onBlur right away. Remove after the issue resolved.
+     */
+    const isFirefox = typeof InstallTrigger !== 'undefined';
```

Comment:
I can verify the issue is fixed!

I have one question about this line. I'm not sure why we are using `InstallTrigger` to check if it's FF.
Can we use `navigator.userAgent` to check the type of browser? Also, could you move this to `device.js`, just like `isIE11`. Thanks!

Reply:
Thank you! Resolved in a2be101.